From 1fcfda5f1dc5675afe54846ca0d01e32b39fbac2 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca.weiss@fairphone.com>
Date: Fri, 1 Mar 2024 10:33:23 +0100
Subject: [PATCH 5/5] uim: Add "File Control Information" to Open Logical
 Channel request

According to gobi-api sources this TLV exists:
https://gitlab.freedesktop.org/mobile-broadband/libqmi/-/blob/main/gobi-api/GobiAPI_2013-07-31-1347/GobiConnectionMgmt/GobiConnectionMgmtAPIStructs.h#L24641-24645
---
 data/qmi-service-uim.json | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/data/qmi-service-uim.json b/data/qmi-service-uim.json
index 181f0418..cd7ca237 100644
--- a/data/qmi-service-uim.json
+++ b/data/qmi-service-uim.json
@@ -1033,7 +1033,13 @@
                      "since"              : "1.36",
                      "format"             : "array",
                      "size-prefix-format" : "guint8",
-                     "array-element"      : { "format" : "guint8" } } ],
+                     "array-element"      : { "format" : "guint8" } },
+                   { "name"          : "File Control Information",
+                     "id"            : "0x11",
+                     "type"          : "TLV",
+                     "since"         : "1.36",
+                     "format"        : "guint8",
+                     "public-format" : "QmiUimFileControlInformation" } ],
      "output"  : [ { "common-ref" : "Operation Result" },
                    { "name"          : "Channel ID",
                      "id"            : "0x10",
-- 
2.45.1

